internal/repository/certificate_repository.go

```diff
@@ -62,3 +64,36 @@ func (profileStore *CertificateStore) CreateCertificate(ctx context.Context, val
 
 	return nil
 }
+
+// GetCertificates fetches certificates details from the database.
+func (certificateStore *CertificateStore) GetCertificatesList(ctx context.Context, profileID int) (values []dto.CertificateResponse, err error) {
```

Comment:
`GetCertificatesList` is GET or List? it seems both.

as per your response type, lets make it `ListCertificates`

Reply:
kk Done